### PR TITLE
Update tox/travis configuration

### PR DESCRIPTION
**Pull request by yakky**
Fix support for Django 1.7
Add Django 1.8 to test matrix
Add explicit django CMS version to tests
Removed special cases from setup.py

### CODE REVIEW

**Comment by yakky** [comment]
Ping @intellectronica @mkoistinen for review

File: requirements-test.txt [hunk 1]
```diff
@@ -6,5 +6,5 @@ nose>=1.3.0
 django-nose>=1.2
 flake8>=2.1.0
 tox>=1.7.0
```
Comment (mkoistinen) [inline requirements-test.txt]: I see we're including Tox, but we're not really using it. I suggest we fully embrace Tox!

File: .travis.yml [hunk 2]
```diff
@@ -12,8 +12,15 @@ sudo: false
 
 env:
   matrix:
-  - DJANGO='django16'
-  - DJANGO='django17'
+  - DJANGO='django16' CMS='cms30'
```
Comment (mkoistinen) [inline .travis.yml]: These should all be tox-envs, no?

File: tox.ini [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py{26,27,33,34}-django16, py{27,33,34}-django17
+envlist=py{26,27,33,34}-django16-cms{30,31,32'}, py{27,33,34}-django{17,18}-cms{31,32}
```
Comment (mkoistinen) [inline tox.ini]: I'd have written this like:
````
py{34,33,27}-django{18,17}-cms{32,31}, py{34,33,27,26}-django16-cms{32,31,30}
````

With this, you can (mostly) generate your travis env-list with `tox -l`

Also, you have a typo. There's a single quote in `cms{30, 31, 32'}`

**Comment by mkoistinen** [comment]
If you can fix the typo in the tox env list then LGTM =)

**Comment by yakky** [comment]
@mkoistinen done

**Pull request merged**